src/crimson/mon/MonClient.h

```diff
@@ -75,6 +75,7 @@ class Client : public ceph::net::Dispatcher,
 
 public:
   Client(ceph::net::Messenger&, ceph::common::AuthHandler&);
+  Client(Client&&);
```

Comment:
OK, thought it was intentionally removed.

Reply:
it was. i thought that CephContext should not be movable. but, well, it's crimson-osd where we use `CephContext` as a normal class, instead of a heavy-weighted catch-all facility holder. that's why i added the move ctor for `CephContext` so `mon::Client` can be movable again.

Reply:
:+1: